iocextractor/core/server.py

```diff
@@ -235,7 +235,8 @@ def handle_scraper_feed(data):
                 iocs = Extractor.extract_ioc(json_data.get('content'))
                 input_filename = sanitize_title(unsanitized_title=str(json_data.get('title')), servicename=SERVICENAME)
                 iocs['input_filename'] = input_filename
-                Extractor.pushfindings(iocs)
+                if iocs:
```

Comment:
This should not work due to the fact that the dict it self is never empty or in this case not none.

Reply:
You should walk over the data structure and check for empty lists and dicts in the dict.

Reply:
True. I chose the wrong position. But if I am placing the if statement directly after line 235 (iocs = Extractor.extract_ioc(json_data.get('content'))) we should be able to check, if the dict is empty or not, because the .extract_ioc() function is returning an empty dict, if no iocs are found right?

Suggestion:

`        try:
            if (json_data := json.loads(data.value.decode("utf-8"))) is not None:
                iocs = Extractor.extract_ioc(json_data.get('content'))
                if iocs:
                    input_filename = sanitize_title(unsanitized_title=str(json_data.get('title')), servicename=SERVICENAME)
                    iocs['input_filename'] = input_filename
                    Extractor.pushfindings(iocs)
        except Exception as error:
            LogMessage(str(error), LogMessage.LogTyp.ERROR, SERVICENAME).log()
`

Reply:
Yes could work but you have to check for "iocs is not None and len(ioc) > 0"

Reply:
Have you tried it?

Reply:
On it.

Reply:
I pushed the changes, but please do not merge it yet. I want to check, if it also runs on our testing system, but for this I had to push the changes.
I will inform you, when the merge is possible.

Reply:
Should be fine.